source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -43,6 +43,28 @@ namespace PlayFabUnit
     constexpr int CLOUDSCRIPT_TIMEOUT_MS = 10000;
     constexpr int CLOUDSCRIPT_TIMEOUT_INCREMENT = 100;
 
+    void PlayFabEventTest::OnErrorSharedCallback(const PlayFab::PlayFabError& error, void* customData)
+    {
+        TestContext* testContext = reinterpret_cast<TestContext*>(customData);
```

Comment:
nit: prefer static_cast

Reply:
I'm going to by design this one.
It's the test framework, not the SDK, so performance isn't nearly as big of a factor (not zero, but...).
And, tests need to be a little more lenient/flexible by their nature of "anticipating incorrect code and or failures".

---
In reply to: [356223570](https://github.com/PlayFab/XPlatCoreTemplate/pull/62#discussion_r356223570) [](ancestors = 356223570)

Reply:
lol. Youu actually used static_cast in this same PR on line 64.

Reply:
After in-person discussion, I think I'll just file a separate task to re-evaluate every reinterpret_cast and try and convert them all to static_cast.

---
In reply to: [356244857](https://github.com/PlayFab/XPlatCoreTemplate/pull/62#discussion_r356244857) [](ancestors = 356244857,356223570)

Reply:
hmm